assessment_software/src/app/app.component.html

```diff
@@ -1,31 +1,41 @@
 <p>Hello World</p>
+<app-datepicker variant="datetime"></app-datepicker>
+<app-datepicker variant="date"></app-datepicker>
 <app-course-card
+  (onCourseCardButtonClicked)="courseCardButton($event)"
+></app-course-card>
```

Comment:
why was another course card component added here?

Reply:
Ohh I think it is when resolved merge conflict.